From 0bea36b16122898d96b3df22b3cae3ad5631440d Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Sat, 2 Dec 2023 17:04:50 +0100
Subject: [PATCH 0803/1141] media: dcmipp: pixelcap: correct height size
 alignment of pixelcap

Constraints only refer to the line rather than the height of the image.

Change-Id: I426fb9c83c6257ee9738f69c6159ba5e1a36e18a
Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/344611
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
---
 drivers/media/platform/st/stm32/stm32-dcmipp/dcmipp-pixelcap.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/media/platform/st/stm32/stm32-dcmipp/dcmipp-pixelcap.c b/drivers/media/platform/st/stm32/stm32-dcmipp/dcmipp-pixelcap.c
index 7a1968a0e8d8..6cfc7acc101c 100644
--- a/drivers/media/platform/st/stm32/stm32-dcmipp/dcmipp-pixelcap.c
+++ b/drivers/media/platform/st/stm32/stm32-dcmipp/dcmipp-pixelcap.c
@@ -409,8 +409,7 @@ static int dcmipp_pixelcap_try_fmt_vid_cap(struct file *file, void *priv,
 			      hdw_pixel_alignment(format->pixelformat),
 			      &format->height,
 			      DCMIPP_FRAME_MIN_HEIGHT, DCMIPP_FRAME_MAX_HEIGHT,
-			      hdw_pixel_alignment(format->pixelformat),
-			      0);
+			      0, 0);
 	if (format->width != in_w || format->height != in_h)
 		dev_dbg(vcap->dev,
 			"resolution updated: %dx%d -> %dx%d\n",
-- 
2.39.2

